Servers/utils/assessment.utils.ts

```
@@ -44,23 +44,26 @@ export const getAssessmentByProjectIdQuery = async (
 
 export const createNewAssessmentQuery = async (
   assessment: Assessment,
-  enable_ai_data_insertion: boolean
+  enable_ai_data_insertion: boolean,
+  transaction: Transaction
 ): Promise<Object> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)